tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -0,0 +1,60 @@
+import { IConfiguration } from "@microsoft/applicationinsights-web";
+export interface ISnippetConfig {
+    src: string;
+    name?: string;
+    ld?: number;
+    useXhr?: boolean;
+    crossOrigin?: string;
+    onInit?: any;
+    cfg: {
+        connectionString: any;
+    };
+}
+
+export interface AIConfig extends IConfiguration {
+    autoExceptionInstrumented?: boolean;
+    url?: string;
+}
+
+export interface Fields {
+    strIngestionendpoint?: any;
+    endpointsuffix?: any;
+    location?: any;
+    // Add other properties as needed
+  }
+
+export interface Envelope{
```

Comment:
We should be able to import these from the core instance rather than redefining here -- I don't remember which file.